packages/@sanity/color-input/src/ColorPicker.js

```diff
@@ -4,41 +4,47 @@ import Button from 'part:@sanity/components/buttons/default'
 import {ColorWrap, Checkboard, Saturation, Hue, Alpha} from 'react-color/lib/components/common'
 import ColorPickerFields from './ColorPickerFields'
 import TrashIcon from 'part:@sanity/base/trash-icon'
+import invertColor from 'invert-color'
```

Comment:
This doesn't seem to be in use?

Reply:
Yeah. Tried another approach first. Left something behind :)